sync_telegram/models/sync_project.py

```diff
@@ -48,12 +51,47 @@ def _eval_context_telegram(self, secrets, eval_context):
         else:
             raise Exception("Telegram bot token is not set")
 
+        def _url2base64(url):
+            r = requests.get(url)
+            datas = base64.b64encode(r.content)
+            return datas
+
         def sendMessage(chat_id, *args, **kwargs):
             log_transmission(
-                "Message to %s@telegram" % chat_id, json.dumps([args, kwargs])
+                "Message sent to %s@telegram" % chat_id, json.dumps([args, kwargs])
```

Comment:
```suggestion
                "%s@telegram" % chat_id, "Message: %s" % ([args, kwargs])
```

Первый аргумент на название цели.  Не нужно в виде предложения делать. Уточнения можно вторым аргументом делать